engine/apps/api/serializers/alert_receive_channel.py

```diff
@@ -376,7 +380,24 @@ def validate(self, data):
                     {"verbal_name": "An integration with this name already exists for this team"}
                 )
 
-        return validated_data
+    def get_team_for_name_validation(self, validated_data):
+        """
+        get_team_for_name_validation retrieves the team to be used in the validation process.
+
+        If the serializer is used to update an existing object, it returns the team from the request data if it's present,
+        otherwise, it returns the team from the existing instance.
+        It's needed to validate name correctly even if team is not present in request data (It's not required).
+
+        If the serializer is used to create a new object, it returns the team from the request data.
+        """
+        if self.instance:
+            if "team" in validated_data:
+                team = validated_data.get("team", None)
+            else:
+                team = self.instance.team
```

Comment:
Could this if block be simplified to
`team = validated_data.get("team", self.instance.team)`
? (same below)

Reply:
Hm, it seems so. For some reason I though that if team is in validated data, but it's none (if integration is un-attached from team for example) it will use second argument of get, but it works!

Reply:
It seems it could be simplified only to   
`team = validated_data.get("team", self.instance.team) if self.instance else validated_data.get("team")`
becase validate is used both for create when self.instance is None and update when self.instance is object being updated.